### PR TITLE
Increase the number of results when using Vomnibar

### PR DESCRIPTION
**Pull request by onriv**
increase the number to 100 by providing 

```css
#vomnibar li.vomnibarHided {
  display: none;
}
```

and calc the range that should be shown

### CODE REVIEW

**Comment by onriv** [comment]
relative  to https://github.com/philc/vimium/issues/2708

File: pages/vomnibar.coffee [hunk 1]
```diff
@@ -96,9 +96,25 @@ class VomnibarUI
       @input.value = @previousInputValue
       @previousInputValue = null
 
-    # Highlight the selected entry, and only the selected entry.
+    # Highlight the selected entry, and only the selected entry, and hide the entries too far away from the
+    # selected
+    [lowest, highest] = @calcRangeForVisibleChildren()
     for i in [0...@completionList.children.length]
-      @completionList.children[i].className = (if i == @selection then "vomnibarSelected" else "")
+      className = (if i == @selection then "vomnibarSelected" else if (lowest <= i < highest) then "" else "vomnibarHided")
```
Comment (philc) [inline pages/vomnibar.coffee]: This inline if statement is now complex and it should be unfolded onto multiple lines:

```
      className =
        if i == @selection then "vomnibarSelected"
        else if (lowest <= i < highest) then ""
        else "vomnibarHided"
```

File: pages/vomnibar.coffee [hunk 2]
```diff
@@ -96,9 +96,25 @@ class VomnibarUI
...
+      @completionList.children[i].className = className
+
+  calcRangeForVisibleChildren: () ->
```
Comment (philc) [inline pages/vomnibar.coffee]: A better name for this is `visibleChildrenRange`. Also add a function-level comment.

File: pages/vomnibar.css [hunk 3]
```diff
@@ -132,6 +132,10 @@
   font-weight: normal;
 }
 
+#vomnibar li.vomnibarHided {
```
Comment (philc) [inline pages/vomnibar.css]: The name of this class should be `vomnibarHidden`.

File: pages/vomnibar.coffee [hunk 4]
```diff
@@ -96,9 +96,25 @@ class VomnibarUI
...
+    visibleRange = 10
+    if @completionList.children.length <= visibleRange
+      highest = @completionList.children.length
+      lowest = 0
+    else
+      highest = @selection+visibleRange//2+1
```
Comment (philc) [inline pages/vomnibar.coffee]: Let's use `math.floor` rather than Coffeescript's obscure syntax. I didn't know what this was and had to look it up.

**Comment by philc** [comment]
Closing in light of the discussion on #2708.

**Pull request closed without merging**